src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -100,17 +100,31 @@ internal static class IntellisenseQuickInfoBuilder
             // build text for RelatedSpan
             if (quickInfoItem.RelatedSpans.Any() && context?.Document is Document document)
             {
+                const string RelatedSpanSeparatorClassification = nameof(RelatedSpanSeparatorClassification);
                 var classifiedSpanList = new List<ClassifiedSpan>();
-                foreach (var span in quickInfoItem.RelatedSpans)
+                for (var i = 0; i < quickInfoItem.RelatedSpans.Length; i++)
                 {
+                    var span = quickInfoItem.RelatedSpans[i];
                     var classifiedSpans = await ClassifierHelper.GetClassifiedSpansAsync(document, span, cancellationToken).ConfigureAwait(false);
                     classifiedSpanList.AddRange(classifiedSpans);
+                    var lastSpan = i == quickInfoItem.RelatedSpans.Length - 1;
+                    if (!lastSpan)
+                    {
+                        // If there is more than one relatedSpan, the single spans need to be separated.
+                        // This marker is used to track, where the separator needs to be inserted.
+                        classifiedSpanList.Add(new ClassifiedSpan(span, RelatedSpanSeparatorClassification));
```

Comment:
is this still used in this pr?  when do you have more htan one related span?

Reply:
That's when you have `#if #elif #endi$$f` I reintroduced to show the whole conditionals today. This change needs to be verified by you again, because I can't F5 and screenshot it.